tests/test_basic.py

```diff
@@ -2,6 +2,7 @@
 
 import os
 
+import hist
```

Comment:
Gotta add it to test reqs as well https://github.com/scikit-hep/mplhep/blob/master/setup.py#L16